R/aalen_johansen.R

```diff
@@ -0,0 +1,114 @@
+#' Aalen Johansen Estimator
+#'
+#' This function calculates the Aalen Johansen estimator of
+#' adverse events  observed in `[0, tau]`.
+#' Please also refer to  \insertCite{stegherr_meta_analytic_2021;textual}{savvyr}.
+#'
+#' Estimating and comparing adverse event probabilities in the presence of varying follow-up times and competing events
+#' Regina Stegherr, Claudia Schmoor, Michael Lübbert, Tim Friede, Jan Beyersmann
+#' First published: 18 May 2021 https://doi.org/10.1002/pst.2130
```

Comment:
this should be moved to the BibTeX file?

Reply:
done